department-of-veterans-affairs/vets-website
# Fix for supply reorder widget

## Summary

The supply reorder page requires the ctaWidget to display a `h4` heading, and not a `h3` heading.

- Adds optional `headingLevel` prop to `src/applications/static-pages/cta-widget/ctaWidgets.js`
- Updates unit specs

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#79823
- _Link to previous change of the code/bug (if applicable)_
department-of-veterans-affairs/vets-website#29499

## Testing done

Steps to verify in local development:

1. `yarn build` from `vets-website`
1. `yarn watch` from `content-build`
1. `foreman start -m all=1,clamd=0,freshclam=0` from `vets-api` branch `master`
1. Add HTML to `content-build/build/localhost/health-care/order-hearing-aid-batteries-and-accessories/index.html`: `<div data-template="paragraphs/react_widget" class="cta-widget" data-widget-type="cta" data-app-id="ha-cpap-supplies"></div>` Note that this step is done through the Drupal content editor for other environments.
1. Visit: `http://localhost:3002/health-care/order-hearing-aid-batteries-and-accessories/#how-do-i-order-hearing-aid-bat`, observe that no content has changed.
1. Visit: `http://localhost:3000/flipper/features/ha_cpap_supplies_cta` and click "Enable for everyone"
1. Visit: `http://localhost:3002/health-care/order-hearing-aid-batteries-and-accessories/#how-do-i-order-hearing-aid-bat`, observe that content _has_ changed.

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Desktop |    ![Screenshot 2024-05-13 at 14-56-00 Order hearing aid batteries and accessories from VA Veterans Affairs](https://github.com/department-of-veterans-affairs/vets-website/assets/1299034/1d956029-2545-445c-9f8b-2205a04f1f36)    |    ![Screenshot 2024-05-13 at 14-55-20 Order hearing aid batteries and accessories from VA Veterans Affairs](https://github.com/department-of-veterans-affairs/vets-website/assets/1299034/78195a08-c4d0-4964-9912-6bfa4ec144ea)   |

## Acceptance criteria

### Quality Assurance & Testing
